src/Workspaces/CSharp/Portable/Formatting/FormattingHelpers.cs

```diff
@@ -73,12 +73,14 @@ public static bool IsCloseParenInParameterList(this SyntaxToken token)
 
         public static bool IsOpenParenInArgumentList(this SyntaxToken token)
         {
-            return token.Kind() == SyntaxKind.OpenParenToken && token.Parent.Kind() == SyntaxKind.ArgumentList;
+            return token.Kind() == SyntaxKind.OpenParenToken &&
+                (token.Parent.IsKind(SyntaxKind.ArgumentList) || token.Parent.IsKind(SyntaxKind.AttributeArgumentList));
         }
```

Comment:
@Pilchie The change I made was for `node.Parent` and that has a potential for Parent being a null. This is `token.Parent` which will always be parented. So dont need to perform that check.

I prefer [`IsKind(...)`](http://index/#Microsoft.CodeAnalysis.CSharp/CSharpExtensions.cs,27) since there is just one check rather than [`IsParentKind(...)`](http://index/#Microsoft.CodeAnalysis.CSharp.Workspaces/Extensions/SyntaxNodeExtensions.cs,19) which does a null check and the same check as `IsKind()`